.rubocop.yml

```
@@ -29,7 +29,7 @@ AllCops:
     - 'tmp/**/*'
     - 'vendor/**/*'
     - 'public/**/*'
-  TargetRubyVersion: 3.3.0
+  TargetRubyVersion: 3.4.0
```

I wonder if we should use `3.4` (general target of 3.4.x) or `3.4.1` (specific version) instead?

```suggestion
  TargetRubyVersion: 3.4
```

---

I don't have a strong sense even after reading [the docs](https://docs.rubocop.org/rubocop/compatibility.html), but I'm less worried about the specific patch version, so I think `3.4` is fine?